recipes/inifile/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "inifile" %}
+{% set version = "0.3" %}
+{% set hash_type = "sha256" %}
+{% set hash = "d0844a8db43f98ed514389b42b3e04243b5f694737feb24ce73906f6d928ed7d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - inifile
+
+about:
+  home: https://github.com/mitsuhiko/python-inifile
+  license: BSD 3-Clause
```

Comment:
Could we please ask upstream to include a `MANIFEST.in` that lists the license file so it is included in packages (e.g. `sdist`s)?